yazi-plugin/preset/plugins/file.lua

```diff
@@ -23,8 +23,8 @@ function M:spot(job)
 			:area(ui.Pos { "center", w = 60, h = 20 })
 			:row(1)
 			:col(1)
-			:col_style(ui.Style():fg("blue"))
-			:cell_style(ui.Style():fg("yellow"):reverse())
+			:col_style(THEME.spot.values)
+			:cell_style(THEME.spot.selected)
```

Comment:
Need help here, as described in https://github.com/sxyazi/yazi/issues/1986#issuecomment-2517850417. This doesn't seem to work. The spotter does not open and there's no messages in the log file.

Reply:
Ah I see what's the problem now, I'll address this in another PR

Reply:
Thank you

Reply:
> Ah I see what's the problem now, I'll address this in another PR

Do we have an issue or any item that we can use to track this dependency? Don't mean to rush. This is a minor cosmetics.

Reply:
We can use https://github.com/sxyazi/yazi/issues/1986 to track this - don't close it after we merge this PR